lib/message.rb

```diff
@@ -0,0 +1,28 @@
+require 'pg'
+require './lib/database_connection.rb'
+
+class Message
+
+  attr_reader :text, :user_name, :id, :time_stamp
+
+  def initialize(text, user_name, id, time_stamp)
+    @text = text
+    @user_name = user_name
+    @id = id
+    @time_stamp = time_stamp
+  end
```

Comment:
Love that the variable naming is logical throughout - keep doing that in future projects. 👍🏻 

It makes the code easy to read and change.